.github/.goreleaser-edge.yml

```diff
@@ -38,7 +38,7 @@ dockers: # https://goreleaser.com/customization/docker/
     goarch: amd64
     dockerfile: Dockerfile-ubi
     image_templates:
-      - "mondoo/{{ .ProjectName }}:edge-{{ .Version }}-ubi-amd64"
+      - "ghcr.io/mondoohq/{{ .ProjectName }}:edge-{{ .Version }}-ubi-amd64"
       - "mondoo/{{ .ProjectName }}:edge-latest-ubi-amd64"
```

Comment:
do we need to update the ubi container as well?

Reply:
we do. Nice catch!